src/He.PipelineAssessment.UI/Features/Workflow/SaveAndContinue/SaveAndContinueCommandValidator.cs

```diff
@@ -0,0 +1,43 @@
+﻿using FluentValidation;
+
+namespace He.PipelineAssessment.UI.Features.Workflow.SaveAndContinue
+{
+    public class SaveAndContinueCommandValidator : AbstractValidator<SaveAndContinueCommand>
+    {
+
+        public SaveAndContinueCommandValidator()
+        {
+            RuleFor(x => x.Data.QuestionActivityData!.MultipleChoice).Must(multipleChoice =>
```

Comment:
The command validator here does not filter based on the QuestionActivityData.ActivityType property - as such it blindly checks all queries, and as such hits issues when certain fields (in this instance SelectedChoices) is null.